test/token/ERC20/BurnableToken.behaviour.js

```diff
@@ -0,0 +1,40 @@
+import assertRevert from '../../helpers/assertRevert';
+
+export default function ([owner]) {
+  describe('basic burnable token', function () {
+    const from = owner;
+
+    describe('when the given amount is not greater than balance of the sender', function () {
+      const amount = 100;
+
+      it('burns the requested amount', async function () {
+        await this.token.burn(amount, { from });
+
+        const balance = await this.token.balanceOf(from);
+        assert.equal(balance, 900);
```

Comment:
This magic 900 here can break because the behavior doesn't know the initial balance. How can we fix this? I think we should include the `beforeEach` with the initial balance here.

Reply:
I added `initialBalance` as a parameter to the behaviour